src/main/java/msjava2lesson/reverselinkedlist/ListNode.java

```diff
@@ -0,0 +1,19 @@
+package msjava2lesson.reverselinkedlist;
+
+public class ListNode{
+
+    public int val;
+
+    public ListNode next;
+
+    ListNode(){}
```

Comment:
why no access modifier here?

Reply:
It is default structure! I do not use this method at all. Removed.